Reliability/300_Testing_for_Resiliency_of_EC2_RDS_and_S3/Code/CloudFormation/lambda_functions_for_deploy.json

```diff
@@ -180,26 +193,26 @@
       "Type" : "AWS::IAM::ServiceLinkedRole",
       "Condition" : "CreateRDSServiceRole",
       "Properties" : {
-        "AWSServiceName" : "rds.amazonaws.com",
-        "Description"    : "This should allow an RDS instance to be created by this account"
+        "AWSServiceName": "rds.amazonaws.com",
+        "Description": "This should allow an RDS instance to be created by this account"
       }
     },
 
     "ELBServiceRole" : {
-      "Type" : "AWS::IAM::ServiceLinkedRole",
-      "Condition" : "CreateELBServiceRole",
-      "Properties" : {
-        "AWSServiceName" : "elasticloadbalancing.amazonaws.com",
-        "Description"    : "This should allow an ELB to be created by this account"
+      "Type": "AWS::IAM::ServiceLinkedRole",
```

Comment:
Is the only change here to remove the spaces before the colon?  I see that done on other changes too, but there are still plenty of places that were not updated (and still include the spaces).  Why fix here (if no other changes) but not elsewhere?  Or we can make the whole thing moot with https://github.com/setheliot/aws-well-architected-labs/issues/16 :-)

Reply:
Yea - part of working on code is cleaning it. Updating parts that I touch only.